src/socket/address.cr

```diff
@@ -239,7 +239,7 @@ class Socket
     end
 
     private def ipv6_addr8(addr : LibC::In6Addr)
-      {% if flag?(:darwin) || flag?(:openbsd) || flag?(:freebsd) || flag?(:dragonfly) %}
+      {% if flag?(:darwin) || flag?(:bsd) %}
```

Comment:
Removing dragonfly?

Reply:
It's covered by the `bsd` flag.

Reply:
flag bsd includes dragonfly:

https://github.com/crystal-lang/crystal/pull/9360/files#diff-0817fdb7bea983f15d454e56956e8725R98